security-analysis/src/main/java/com/powsybl/security/SecurityAnalysisTool.java

```diff
@@ -156,7 +156,7 @@ public void run(CommandLine line, ToolRunningContext context) throws Exception {
             JsonSecurityAnalysisParameters.update(parameters, parametersFile);
         }
 
-        SecurityAnalyzer analyzer = new SecurityAnalyzer(limitViolationFilter, context.getComputationManager(), 0, interceptors);
+        SecurityAnalyzer analyzer = new SecurityAnalyzer(limitViolationFilter, context.getShortTimeExecutionComputationManager(), 0, interceptors);
```

Comment:
Change to getLongTimeExecutionComputationManager

Reply:
Done